.github/workflows/build_external_container_images.yaml.yml

```diff
@@ -0,0 +1,130 @@
+name: Build Bitnami Container Images
+
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+  packages: write
+
+jobs:
+  build_and_push_images:
+    name: Build and Push ${{ matrix.image.name }} Image
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        image:
+          # On Ref we use the specific commit SHA to ensure reproducible builds
+          # It can be: branch, tag or SHA
+          - name: PostgreSQL
+            image_name: chainloop-dev/chainloop/postgresql
+            path: bitnami/postgresql/16/debian-12
+            sparse_checkout: bitnami/postgresql/16/debian-12
```

Comment:
it might be useful to add a comment here with the version that's targetting to build